tutorial/custom-cache-ttl.md

```diff
@@ -7,12 +7,36 @@ nav_exclude: true
 
 # {{page.title}}
 
-_This tutorial was contributed by [Amogh](https://github.com/ags1773)_
+_This tutorial was contributed by [Amogh](https://github.com/ags1773)_ and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/) 
 
-The time for which resources are cached on CDN i.e. the s-maxage value in the Cache-Control header is set to 900 seconds for most requests. This can be changed if needed from the app level.
+TTL is nothing but the time for which resources are cached on CDN. The value `s-maxage` is stored in a part of the response called the HTTP header, and it specifies for how many seconds, minutes, or hours content will be cached. In this tutorial, we are going to discuss how to override  `s-maxage` value in response cache-control header for `isomorphicRoutes` and `upstreamQuintypeRoutes`.  
 
-- Make sure [@quintype/framework](https://www.npmjs.com/package/@quintype/framework) version is `4.16.2` or higher
+#### How to override  s-maxage value in Isomorphic Routes
 
+For overriding the s-maxage value, We can set `isomorphicRoutesSmaxage: <value>` under publisher in publisher.yml config file that comes from BlackKnight or pass it from the app level.  The default value is set to 900 seconds.
+
+- Make sure [@quintype/framework](https://www.npmjs.com/package/@quintype/framework) version is `5.0.5` or higher
+
+**How to pass it from Black Knight **
+
+Go to Black Knight  `/app/config/publisher.yml`,  add `isomorphicRoutesSmaxage:  <value>` under `publisher`.
```

Comment:
Remove space from Black Knight. And add this URL https://black-knight.quintype.com/

Reply:
done